src/Shared/AssemblyInfo.Common.cs

```diff
@@ -1,15 +0,0 @@
-﻿// Copyright (c) Six Labors and contributors.
-// Licensed under the Apache License, Version 2.0.
-
-using System.Reflection;
-using System.Resources;
-using System.Runtime.CompilerServices;
-
-// Ensure the internals can be built and tested.
-[assembly: InternalsVisibleTo("SixLabors.ImageSharp.Drawing")]
-[assembly: InternalsVisibleTo("ImageSharp.Benchmarks")]
-[assembly: InternalsVisibleTo("SixLabors.ImageSharp.Tests")]
-[assembly: InternalsVisibleTo("SixLabors.ImageSharp.Sandbox46")]
-
-[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2, PublicKey=0024000004800000940000000602000000240000525341310004000001000100c547cac37abd99c8db225ef2f6c8a3602f3b3606cc9891605d02baa56104f4cfc0734aa39b93bf7852f7d9266654753cc297e7d2edfe0bac1cdcf9f717241550e0a7b191195b7667bb4f64bcb8e2121380fd1d9d46ad2d92d2d15605093924cceaf74c4861eff62abf69b9291ed0a340e113be11e6a7d3113e92484cf7045cc7")]
-[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2, PublicKeyToken=null")]
```

Comment:
Need to have an `InternalsVisibleTo` for this, will figure out how to get that working...

Reply:
Should be fixed.